.github/workflows/update_golang_version.yml

```diff
@@ -46,11 +48,10 @@ jobs:
           echo "bootstrap-version=${bootstrap_version}" >> $GITHUB_OUTPUT
 
           # Check if the PR already exists, if it does then do not create new PR.
-          gh pr list -S "is:open [${{ matrix.branch }}] Upgrade the Golang version to go${go_version}" | grep "OPEN"
```

Comment:
`grep -q` is what you were missing here. IMO that's a better fix.

Reply:
I am not sure `grep -q` works in this case, I have tried it out locally and I still see the same issue as before my fix. The issue is that `grep` exits with `1` since there is no match found and thus the workflow fails.